decklink/src/info.rs

```diff
@@ -91,6 +92,9 @@ impl ProfileAttributes {
             number_of_subdevices: self.get_integer(IntegerAttributeId::NumberOfSubDevices)?,
             subdevice_index: self.get_integer(IntegerAttributeId::SubDeviceIndex)?,
             persistent_id: self.get_integer(IntegerAttributeId::PersistentID)?,
+            persistent_id_hex_str: self
```

Comment:
This struct is only used for debug purposes to print params in example code, so instead overriding debug info to print all those information both hex in dec I opted just to add field.